tvApp/src/main/java/band/effective/office/tv/screen/autoplayMenu/AutoplayMenuScreen.kt

```diff
@@ -47,12 +47,14 @@ import band.effective.office.tv.ui.theme.robotoFontFamily
 data class ItemRes(
     val text: String,
     val icon: Painter? = null,
-    val activeIcon: Painter? = null
+    val activeIcon: Painter? = null,
 ) {}
 
 @Composable
 fun AutoplayMenuScreen(
-    viewModel: AutoplayMenuViewModel = hiltViewModel(), navController: NavController
+    viewModel: AutoplayMenuViewModel = hiltViewModel(),
+    navController: NavController,
```

Comment:
maybe it's better to use `callback` instead of passing `navContoller`

Reply:
it's old logic, refactor is not main target it's pr